src/Sarif/Writers/CachingLogger.cs

```diff
@@ -12,8 +12,12 @@ namespace Microsoft.CodeAnalysis.Sarif.Writers
     /// instances. The driver framework uses this mechanism to merge results
     /// produced by a multi-threaded analysis into a single output file.
     /// </summary>
-    public class CachingLogger : IAnalysisLogger
+    public class CachingLogger : BaseLogger, IAnalysisLogger
     {
+        public CachingLogger(IEnumerable<FailureLevel> level, IEnumerable<ResultKind> kind) : base(level, kind)
```

Comment:
>level [](start = 55, length = 5)

rename to levels and kinds

Reply:
Done.

---
In reply to: [561427374](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r561427374) [](ancestors = 561427374)